app/assets/stylesheets/styles/components/pages/_featured_crops.css.scss

```diff
@@ -0,0 +1,42 @@
+.featured-crops {
+  background-color: rgb(240, 245, 240);
+  padding: 40px inherit 20px;
+
+  .featured-crops-grid {
+    margin-top: 30px;
+
+    .featured-crop-container {
+      margin-bottom: 30px;
+
+      .featured-crop {
+        position: relative;
+        height: 280px;
+
+        &:hover {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 3, but was 4